frontend/views/lobby/lobby.html

```diff
@@ -11,50 +11,52 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
 </head>
 <body>
-    <header>
-        <span>Statki</span>
-    </header>
-
-    <main>
-        <div class="kontener">
-            <div class="lobby">
-                <div class="hover">
-                    <div><a class="gry" href="#">Lobby1 <button class="join1">JOIN</button>
-                        <span class="material-symbols-outlined">
-                            lock
-                        </span>
-                        </a>
+    <div class="app">
+        <header>
+            <span>Statki</span>
+        </header>
+    
+        <main>
+            <div class="kontener">
+                <div class="lobby">
+                    <div class="hover">
+                        <div><a class="gry" href="#">Lobby1 <button class="join1">JOIN</button>
```

Comment:
Ok, I already checked it: https://www.linkedin.com/advice/1/how-do-you-name-html-classes-ids-skills-html it's actually kebab case (so I suggest e.g. join-lobby or join-button)

Reply:
The thing is that these names have been here since the last pull request I only added a div surrounding then with a app class so it seems like they have been added just now.
We already have a ticket assigned to fix it.